akvo/rsr/views/my_rsr.py

```diff
@@ -244,7 +244,7 @@ def project_editor(request, project_id):
     except Project.DoesNotExist:
         return Http404
 
-    if not request.user.has_perm('rsr.change_project', project):
+    if not request.user.has_perm('rsr.change_project', project) or project.status == 'C':
```

Comment:
Superusers and Akvo Admins (```user.is_superuser``` and ```user.is_admin```) should always be able to open the project editor for any project.